org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateResourceShapeJsp.mtl

```diff
@@ -103,11 +103,11 @@ To revert to the default generated content, delete all content in this file, and
 
                     <h2>Describes</h2>
                     <ul>
-                        <%for(URI next : aResourceShape.getDescribes()) {%>
-                            <li>
-                                <a href="<%=next%>">
-                                    <%= next.toString().split("#")['[1]'/]%></a>
-                            </li>
+                        <%for(URI next : aResourceShape.getDescribes()) {
```

Comment:
package.java registers all namespaces and Application.java registers all shapes; can we do a lookup here?

Reply:
Sure! but then we'll be hiding the problem from the modeller. Ins't it better to show testnsResource, so the user realises there is something that is not quite right?

Reply:
If we're not willing to fix it (which was my original suggestion to add "/" or "#") then I think we should not hide it.
And to be clear, I agree now that we should not be fixing it.

Reply:
No, that is not a problem strictly speaking. The rules in RDF and XML are that the Namespace Name and the Local Name are concatenated directly. The Local Name remains to be `Resource` and the Namespace Name is `example.com/testns`. You can still print `testnsResource` but you can't claim that you are printing a Local Name of the RDF resource.

https://www.w3.org/TR/2006/REC-xml-names11-20060816/#concepts

Reply:
Ok. I retract that I am printing Local Name. It's just a html page, with a short representation of the complete url. 
Doing look ups in the Application.java class is not an option really. Just too much logic for a debug page.
Not sure how to proceed really. Only robust solution is to simply print the whole URI all the time.